man/journal-upload.conf.xml

```diff
@@ -129,6 +129,28 @@
         <listitem><para>Takes a boolean value, enforces using compression without content encoding negotiation.
         Defaults to <literal>false</literal>.</para>
 
+        <xi:include href="version-info.xml" xpointer="v258"/></listitem>
+      </varlistentry>
+      <varlistentry>
+        <term><varname>Header=</varname></term>
+
+        <listitem><para>HTTP header which should be added to each request to URL.</para>
+
+        <para>Header name can contains alphanumeric values, "_" and "-" symbols additionally.</para>
+
+        <para>This option may be specified more than once, in which case all listed headers will be set. If
+        the same header name is listed more than once, all its unique values will be concatenated with comma.
```

Comment:
Please mention that an empty string clears all previous assignments.
Also, please mention that the header name and value must be separated with ':'.